micro-rdk/src/common/app_client.rs

```diff
@@ -323,6 +329,46 @@ impl AppClient {
         Ok((Box::new(cfg_response), datetime))
     }
 
+    pub async fn get_agent_config(&self) -> Result<Box<DeviceAgentConfigResponse>, AppClientError> {
+        let host_info = Some(HostInfo {
+            platform: "linux/amd64".to_string(),
```

Comment:
since micro-rdk isn't fully plugged into the viam-agent ecosystem, need to fudge a little to get the right information stored in the backend. We could make a ticket for getting this fixed along with our other app-side support tickets.

Reply:
I think I'd like this ticket to be made and linked here, although I suspect the discussion around this to be non-trivial

Reply:
done, APP-7672. noted in comment